boards/posix/native_posix/doc/board.rst

```diff
@@ -366,10 +384,66 @@ If the SW unmasks a pending interrupt while running, or triggers a SW
 interrupt, the interrupt controller may raise the interrupt immediately
 depending on interrupt priorities, masking, and locking state.
 
-Normally the resulting executable runs fully decoupled from the real host time.
-That is, simulated time will advance as fast as it can. This is desirable when
-running in a debugger or testing in batch, but not if one wants to interact
-with external interfaces which are based on the real host time.
+About time in native_posix
+==========================
+
+Normally simulated time advances runs fully decoupled from the real host time
+and as fast as the host compute power would allow.
+This is desirable when running in a debugger or testing in batch, but not if
+one wants to interact with external interfaces which are based on the real
+host time.
+
+The Zephyr kernel is only aware of the simulated time as provided by the
+HW models. Therefore any normal Zephyr thread will also know only about
+simulated time.
+
+The only link between the simulated time and the real/host time, if any,
+is created by the clock and timer model.
+
+This model can be configured to slow down the execution of native_posix to
+real time.
+You can do this with the `--rt` and `--no-rt` options from command line.
```

Comment:
use double back ticks for formatting here:

    ``--rt`` and ``--no-rt`` options ...

There are more occurrences of single back-tick formatting in the rest of the document that should be change to doubles.